src/libs/actions/IOU.ts

```diff
@@ -4984,7 +4980,7 @@ function navigateToNextPage(iou: OnyxEntry<OnyxTypes.IOU>, iouType: string, repo
 
     // If we're adding a receipt, that means the user came from the confirmation page and we need to navigate back to it.
     if (path.slice(1) === ROUTES.MONEY_REQUEST_RECEIPT.getRoute(iouType, report?.reportID)) {
-        Navigation.navigate(ROUTES.MONEY_REQUEST_CONFIRMATION.getRoute(iouType, report?.reportID));
+        Navigation.navigate(ROUTES.MONEY_REQUEST_STEP_CONFIRMATION.getRoute(iouType as ValueOf<typeof CONST.IOU.TYPE>, '1', report?.reportID ?? '1'));
```

Comment:
This change is a temporary fix for the TS error. This function only be used in NewRequestAmountPage. This page will be removed in https://github.com/Expensify/App/pull/35833

So this function also be removed